src/main/java/seedu/duke/lists/WorkoutList.java

```diff
@@ -8,5 +9,35 @@
  * Adding, deleting and other operations related to work out routines will be executed from here.
  */
 public class WorkoutList {
-    public static ArrayList<Workout> workoutList = new ArrayList<>();
+    private ArrayList<Workout> workouts;
```

Comment:
where as I think workouts should be static, since in the entire Duke there will be one single list of workouts.

Reply:
there's a getter for this ArrayList if needed, but most commands don't need the ArrayList to be accessed eg. WorkoutList has `addWorkout` , `removeWorkout` and `getWorkout` so we don't have to worry about manipulating data in the actual ArrayList itslef

Reply:
ah ok got it